packages/db-objects-redis/src/lib/objects/constants.ts

```diff
@@ -0,0 +1,37 @@
+export enum ERRORS {
+    ERROR_PERMISSION = 'permissionError',
+    ERROR_NOT_FOUND = 'Not exists',
+    ERROR_DB_CLOSED = 'DB closed'
+}
+
+export const SYSTEM_ADMIN_USER = 'system.user.admin';
+export const SYSTEM_ADMIN_GROUP = 'system.group.administrator';
+
+export const ACCESS_EVERY_EXEC = 0x1;
+export const ACCESS_EVERY_WRITE = 0x2;
+export const ACCESS_EVERY_READ = 0x4;
+export const ACCESS_EVERY_RW = ACCESS_EVERY_WRITE | ACCESS_EVERY_READ;
+export const ACCESS_EVERY_ALL = ACCESS_EVERY_WRITE | ACCESS_EVERY_READ | ACCESS_EVERY_EXEC;
+
+export const ACCESS_GROUP_EXEC = 0x10;
+export const ACCESS_GROUP_WRITE = 0x20;
+export const ACCESS_GROUP_READ = 0x40;
+export const ACCESS_GROUP_RW = ACCESS_GROUP_WRITE | ACCESS_GROUP_READ;
+export const ACCESS_GROUP_ALL = ACCESS_GROUP_WRITE | ACCESS_GROUP_READ | ACCESS_GROUP_EXEC;
+
+export const ACCESS_USER_EXEC = 0x100;
+export const ACCESS_USER_WRITE = 0x200;
+export const ACCESS_USER_READ = 0x400;
+export const ACCESS_USER_RW = ACCESS_USER_WRITE | ACCESS_USER_READ;
+export const ACCESS_USER_ALL = ACCESS_USER_WRITE | ACCESS_USER_READ | ACCESS_USER_EXEC;
```

Comment:
We could think about turning these into `enum`s

Reply:
Problem is that or logic isn’t working in enums so would have to work around but yes can change 👍

Reply:
Yeah, this is not a priority - just an idea for a future cleanup. The usage sites would have to be updated aswell of course.